src/huggingface_hub/__init__.py

```diff
@@ -108,6 +108,8 @@
         "DatasetSearchArguments",
         "HfApi",
         "ModelSearchArguments",
+        "RefInfo",
```

Comment:
```suggestion
        "GitRefInfo",
        "GitRefs",
```

maybe to make it extra clear that it's git stuff that's exposed?

Reply:
Yes, good idea :+1